src/routes/cohorts/cohorts.controller.js

```diff
@@ -12,7 +12,12 @@ const getCohorts = snapshot => {
     }
   }, {});
 
-  return cohorts;
+  return Object.keys(cohorts)
+    .map(key => ({
+      id: key,
+      applicants: cohorts[key]
+    }))
+    .sort((a, b) => -1 * a.id.localeCompare(b.id)); //returns sorted array in "desc" order
```

Comment:
I don't think this sort compare function will work for all cases. For example, `"cohort-6".localeCompare("cohort-10")` is 1 and `"cohort-6".localeCompare("cohort-8")` is -1